static/src/js/App.jsx

```diff
@@ -60,34 +60,26 @@ const App = () => {
   }, [])
 
   const { graphQlHost } = getApplicationConfig()
-
-  const [cookies] = useCookies(['data'])
-
-  let token
-  const { data } = cookies
-  if (data) {
-    ({ token } = decodeCookie(data))
-  }
+  const { user } = useAppContext()
+  const { token } = user
+  const { tokenValue } = token || {}
 
   const httpLink = createHttpLink({
     uri: graphQlHost
   })
 
-  // TODO remove eslint-disable after this method expands in MMT-3407
-  // eslint-disable-next-line arrow-body-style
-  const authLink = setContext((_, { headers }) => {
-    // TODO MMT-3407 - get a real token from launchpad to send
+  console.log('using token ', tokenValue)
```

Comment:
I'd like to keep this log statement and the other log statement showing the refresh token for a short time just to verify things are working as expected in SIT.